GREEDY/DataManagers/DataConverter.cs

```diff
@@ -19,24 +19,54 @@ public List<Item> ReceiptToItemList(Receipt receipt)
             var receiptLinesToString = String.Join(Environment.NewLine, receipt.LinesOfText);
             List<Item> itemList = new List<Item>();
 
-            if (shop == "RIMI" || shop == "MAXIMA")
+            if (shop == "MAXIMA")
             {
-                string pattern = @"([*]+)\n(.+)\n([*]+)";
+                string pattern = @"\b(([A-Z]|#)(\d){8})(.+)\n(PVM\b)";
                 receiptLinesToString = Regex.Replace(receiptLinesToString, @"\r", "");
 
                 Match match = Regex.Match(receiptLinesToString, pattern, RegexOptions.Singleline);
                 if (match.Success)
                 {
-                    receiptLinesToString = match.Groups[2].Value;
-                    pattern = @"([^..]*)([.]+)( \d+,\d\d)";
+                    var productList = match.Groups[4].Value;
+                    productList = Regex.Replace(productList, @"\n", " ");
+                    productList = Regex.Replace(productList, @"(\d+(,)\d\d).[A|E|B|F|N]{1}\b", "$1" + Environment.NewLine);
 
-                    MatchCollection matches = Regex.Matches(receiptLinesToString, pattern, RegexOptions.Singleline);
+                    pattern = @"([A-Za-z]{2}[A-Za-z]+.+)(\d+(,)\d\d)\r\n";
```

Comment:
Maybe a brief comment on how this regex pattern works?

Reply:
@valentk777

Reply:
sure, thanks